NoNicotine_Business/Handler/GetPatientQueryHandler.cs

```diff
@@ -16,12 +17,10 @@ namespace NoNicotine_Business.Handler
     public class GetPatientQueryHandler : IRequestHandler<GetPatientQuery, Response<Patient>>
     {
 
-        private readonly AppDbContext _context;
-        private readonly ILogger<GetPatientQueryHandler> _logger;
```

Comment:
Why did you remove the logger interface?

Reply:
Because we were not using it... jeje

Reply:
But will we never use it? xd

Reply:
maybe later, but for now nop